### PR TITLE
More docs

### PR DESCRIPTION
**Pull request by mmatera**
This must be the last round of doc fixings. So, I activated the consistency tests.

### CODE REVIEW

**Comment by mmatera** [comment]
What it is missing is to complete documentation  in `pymathics.graph.properties`

**Comment by mmatera** [comment]
@rocky 
This completes the basic documentation of the module. Please have a look to check for grammar errors and improvements in the writting.

File: pymathics/graph/base.py [hunk 1]
```diff
@@ -1519,6 +1521,10 @@ def eval_s_t(self, graph, s, t, expression, evaluation, options):
 
 class FindVertexCut(_NetworkXBuiltin):
     """
+    <url>
+    :WMA:
```
Comment (rocky) [inline pymathics/graph/base.py:1525]: NetworkX link could be added (nx.minimum_node_cut)

File: pymathics/graph/operations.py [hunk 2]
```diff
@@ -18,6 +18,11 @@
 
 class FindSpanningTree(_NetworkXBuiltin):
     """
+    <url>:Spanning Tree:https://en.wikipedia.org/wiki/Spanning_tree</url>
+    (<url>:WMA:
```
Comment (rocky) [inline pymathics/graph/operations.py:22]: NetworkX link would be to `nx.minimum_spanning_edges`

File: pymathics/graph/parametric.py [hunk 3]
```diff
@@ -86,7 +86,7 @@ class BarbellGraph(_NetworkXBuiltin):
     :Barbell graph:https://en.wikipedia.org/wiki/Barbell_graph
     </url> (
     <url>
-    :NetworkX:https://networkx.org/documentation/stable/reference/\
+    :NetworkX:https://networkx.org/documentation/networkx-2.8.8/reference/\
```
Comment (rocky) [inline pymathics/graph/parametric.py:89]: Good - thanks.

File: pymathics/graph/parametric.py [hunk 4]
```diff
@@ -198,6 +198,9 @@ def eval(
 
 class CompleteGraph(_NetworkXBuiltin):
     """
+    <url>
+    :WMA:https://reference.wolfram.com/language/ref/CompleteGraph.html
```
Comment (rocky) [inline pymathics/graph/parametric.py:202]: NetworkX links would be to `nx.complete_graph` and `nx.complete_multiparatie_graph`.

File: pymathics/graph/parametric.py [hunk 5]
```diff
@@ -226,7 +229,12 @@ def eval_multipartite(self, n, evaluation: Evaluation, options: dict):
 
 
 class CompleteKaryTree(_NetworkXBuiltin):
-    """<dl>
+    """
+    <url>
+    :K-ary Tree:https://en.wikipedia.org/wiki/M-ary_tree
```
Comment (rocky) [inline pymathics/graph/parametric.py:234]: NetworkX and WMA links?

File: pymathics/graph/parametric.py [hunk 6]
```diff
@@ -269,10 +277,14 @@ def eval_2(self, n, expression, evaluation: Evaluation, options: dict):
...
-        <dt>'CycleGraph[$n$]'
-        <dd>Returns the cycle graph with $n$ vertices $C_n$.
-      </dl>
+    """
+    <url>:Cycle Graph:
+    https://en.wikipedia.org/wiki/Cycle_graph</url>
+
+    <dl>
+      <dt>'CycleGraph[$n$]'
+      <dd>Returns the cycle graph with $n$ vertices $C_n$.
```
Comment (rocky) [inline pymathics/graph/parametric.py:286]: NetworkX and WMA.

Comment (mmatera) [reply pymathics/graph/parametric.py:286]: Ok, let's do that.

**Comment by rocky** [comment]
LGTM - there are a lot more WMA and NetworkX links that should be added. If you want to do this in another PR. Thanks.

Also needed, you will find 

* Builtins with no examples (EdgeIndex, EdgeList, ...) 
*  The idiom `g = Graph[...]; <Builtin>` would better be `g = Graph` - no semicolon so user can _see_ what is up. And then not repeat it in the next example; `Degree`  and `FindVertexCut are like this, but there are other places. 
* Improper comments. See `Degree` and PageRankCentrality - which also has an extra newline between NetworkX and WMA

**Comment by mmatera** [comment]
Also I found at least one symbol that is duplicated in functionality. Probably we need to do another round, but again, this is just to establish a baseline.

**Pull request merged**